src/desktop/lib/buildServerAppContext.ts

```diff
@@ -25,6 +25,7 @@ export const buildServerAppContext = (
       contextPageOwnerType: pageType,
       contextPageOwnerSlug: pageSlug,
     },
+    featureFlags: res.locals.sd.FEATURE_FLAGS,
```

Comment:
I pass `featureFlags` here to be able to get the flag value from `context` in routing (for example, in `prepareVariables` like [here](https://github.com/artsy/force/blob/92c4c4580bdc432f586e05ba88dc96f7239c4bbd/src/v2/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts#L7))